### PR TITLE
clothing support

### PR DESCRIPTION
**Pull request by bell07**
Implements #15

### CODE REVIEW

File: skin_meta_api.lua [hunk 1]
```diff
@@ -60,6 +60,18 @@ function skin_class:get_preview()
 end
 
 function skin_class:apply_skin_to_player(player)
+
+	local function concat_texture(base, ext)
+		if base == "blank.png" then
+			return ext
+		elseif ext == "blank.png" then
+			return base
+		else
```
Comment (SmallJoker) [inline skin_meta_api.lua]: `else` is redundant when all cases above return.

Comment (bell07) [reply skin_meta_api.lua]: else is for the case the base is not blank.png AND ext is not blank.png

Comment (SmallJoker) [reply skin_meta_api.lua]: In that case `return base .. "^" .. ext` would be executed anyway, because that does not have any condition.

**Comment by SmallJoker** [comment]
Nice work, looks good overall :+1:

File: skin_meta_api.lua (hunk 1 above)
Comment (bell07) [reply skin_meta_api.lua]: Ah, ok ;-) 
I like the "if" / "elseif" chains syntax as is, as replacement for not supported "case" command in lua. I think there is no difference in performance or execustion, just developers coding-stlye taste. So I keep the code as is

**Pull request merged**